### ISSUE TITLE
Le bouton d'aide à la saisie capte la touche `Entrer`

### ISSUE THREAD

**Issue by laem** [datagir/nosgestesclimat-site#951]
Pour la navigation au clavier. Potentiellement un problème qui existait déjà avant ta PR Benjamin, à tester. 

Je l'ai trouvé en diagnostiquant le problème du double compte de "1ère réponse au bilan".

**Comment by bjlaa** [comment]
Étrange ! Je me demande ce qui pourrait générer ce comportement dans les modifs que j'ai apportée 🤔 Pour être sûr qu'on est bien aligné sur le problème détecté : quand le focus est sur le champ de réponse principal et que j'appuie sur "Enter" cela ouvre le formulaire de réponse détaillée, c'est bien ça ?

**Comment by laem** [comment]
Yep. Et valable aussi pour la PR avion. C'était peut-être déjà là !
